learn-express-build-strock-trading-platform/server090.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, function() {
+  console.log('Your app is listening on port 3000.');
+});
+
+app.get('/hi', (req, res) => {
+  res.send('Hi there!');
+});
+
+const prices = {
+  ABC: 48.83,
+  DEF: 2.98,
+  GHI: 3.99,
+  JKL: 99,
+  MNO: 45.38,
+  OPQ: 0.48,
+  RST: 9.32,
+  UVW: 10.94,
+  XYZ: 5.32,
+};
+
+app.get('/buy/:ticker/:shares', (req, res) => {
+  const ticker = req.params.ticker;
+  const shares = req.params.shares;
+
+  const total = shares * prices[ticker];
+
+  res.send(
+    `Transaction complete, you purchased ${shares} shares of ${ticker} at $${
+      prices[ticker]
+    }/share for a total of $${total}.`,
+  );
+});
+
+// 090.  Now, let's create the /sell endpoint, which is structured very similar to the /buy endpoint.  Copy the /buy route that you just created and modify it to a '/sell/:ticker/shares' route.  In the response, be sure to include the ticker, number of shares, price per share, and total amount.
```

Comment:
This needs to be broken down, but since users are familiar with creating endpoints each step doesn't have to be so granular.

The first step could be prompting the user to write
```
app.get('sell/:ticker/:shares');
```

The next could be passing it the callback function. The third could be prompting them to create `ticker`, `shares`, and `prices` variables like before. Then the last would be to use `res.send()` with a similar template string, with just some slight modifications.

Reply:
```suggestion
// test 090.  Now, let's create the /sell endpoint, which is structured very similar to the /buy endpoint.  Copy the /buy route that you just created and modify it to a '/sell/:ticker/shares' route.  In the response, be sure to include the ticker, number of shares, price per share, and total amount.
```

Reply:
Thanks for the suggestions @scissorsneedfoodtoo , I'm finally coming back to this. 
I'll create another branch called 'feat: second draft...' and incorporate your suggestions above manually.

Out of curiosity, how did you get the nice quote formating of the code before each of your comments above?  Do I need to do anything in terms of GitHub workflow in this PR thread?

Reply:
@lmilliken, thank you for taking another pass at this! It's a great project and students will learn a lot going through it. All the content is there, it just needs to be broken down a bit more.

By quote formatting, do you mean the gray blocks of code in some of my comments? If so, I believe that's just GitHub's markdown formatting. For single lines, just wrap your code in single backticks (`), and for multi-line use triple backticks (```). I don't think you need to do anything in particular when it comes to GitHub workflow. Creating another branch and incorporating the suggestions above is totally fine.

Also, do you have about 20 or 30 minutes to hop on a call? It would be great to get to know you a bit better and talk about this and the calorie counter project in more detail. If so, please send me an email at kris@freecodecamp.org so we can set something up!